pulsar-broker-common/src/main/java/com/yahoo/pulsar/broker/ServiceConfiguration.java

```diff
@@ -291,12 +301,32 @@ public void setWebServicePortTls(int webServicePortTls) {
         this.webServicePortTls = webServicePortTls;
     }
 
-    public boolean isBindOnLocalhost() {
-        return bindOnLocalhost;
+    public String getBindAddress() {
```

Comment:
Addressed in https://github.com/yahoo/pulsar/pull/26/commits/37450339cc00d11793ebef16b9a3959526bc8825